index.html

```diff
@@ -7,7 +7,7 @@
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/atom-one-dark.min.css">
 	<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/jsoneditor/5.19.0/jsoneditor.min.css" />	
 	<link rel="stylesheet" href="general.css">
-	<link rel="stylesheet" href="decorate.css">
+	<style id="decoratorStylesheet" href="decorate.css"></style>
```

Comment:
We fetch decorate.css via AJAX (see [changes ](https://github.com/ZS/powergrid/pull/42/files#diff-96002143770e23013559d670d3b53871R350) in demo.js) and dump it's contents here. This way we can show it's contents in full source tab.